src/main/java/ch/uzh/ifi/hase/soprafs22/controller/ChatController.java

```diff
@@ -10,24 +9,29 @@
 import ch.uzh.ifi.hase.soprafs22.rest.dto.chat.ChatOverViewGetDTO;
 import ch.uzh.ifi.hase.soprafs22.rest.mapper.DTOMapper;
 import ch.uzh.ifi.hase.soprafs22.service.ChatService;
+import ch.uzh.ifi.hase.soprafs22.service.MatchService;
 import ch.uzh.ifi.hase.soprafs22.service.UserService;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.ArrayList;
+import java.util.Iterator;
 import java.util.List;
 
+
 @RestController
 public class ChatController {
 
   private final UserService userService;
+  private final MatchService matchService;
```

Comment:
What is the reason behind having a MatchService? UserService could do the same things. (Question)

Reply:
added to userService, deleted matchService. done